configs/env-example

```diff
@@ -9,7 +9,7 @@ BUILD_ID=0
 API_PORT=3000
 APP_PORT=8080
 SWAGGER_ENABLED=true
-APPLICATION_ARCHIVE_DAY=43
+APPLICATION_ARCHIVE_DAYS=43
```

Comment:
Adding the comment here for the lack of a better place.
The env variable `APPLICATION_ARCHIVE_DAYS` must be moved from the `api-deploy.yml` to the `queue-consumers-deploy.yml`.